numaprom/_constants.py

```diff
@@ -44,6 +44,36 @@
             "namespace_hash_pod_http_server_requests_latency",
         ],
     },
+    "fuzzy_argocd": {
+        "name": "fuzzy_argocd",
+        "win_size": 12,
+        "threshold_min": 0.1,
+        "model_name": "ae_sparse",
+        "retrain_freq_hr": 8,
+        "resume_training": "True",
+        "num_epochs": 100,
+        "keys": ["namespace", "name"],
+        "scrape_interval": 5,
+        "metrics": [
+            "namespace_http_numalogic_o11yfuzzygqlfederation_segment_api_latency",
+            "namespace_http_numalogic_o11yfuzzygqlfederation_segment_api_error_count",
+        ],
+    },
+    "fuzzy_rollouts": {
+        "name": "fuzzy_rollouts",
+        "win_size": 12,
+        "threshold_min": 0.001,
+        "model_name": "ae_sparse",
+        "retrain_freq_hr": 8,
+        "resume_training": "True",
+        "num_epochs": 50,
+        "keys": ["namespace", "name"],
```

Comment:
this should have `hash_id` right?

Reply:
No, the keys wouldn't have hash id, these are for training. I should may be rename it training_keys?

Reply:
Maybe; Otherwise might get confusing